modin/core/execution/utils.py

```diff
@@ -29,3 +31,45 @@ def set_env(**environ):
     finally:
         os.environ.clear()
         os.environ.update(old_environ)
+
+
+# Check if the function already exists to avoid circular imports
+if "remote_function" not in dir():
+    from modin.config import Engine
+
+    if Engine.get() == "Ray":
+        from modin.core.execution.ray.common import RayWrapper
+
+        _remote_function_wrapper = RayWrapper.put
```

Comment:
```suggestion
        _preprocess_func = RayWrapper.put
```
Maybe we should name this by analogy with the partition_mgr.preprocess_func?

Reply:
Renamed and added checks for the default values. A nested function may not capture objects from the outer scope, but each instance may have different default attributes.